src/apps/search-header/search-header.test.ts

```diff
@@ -0,0 +1,213 @@
+                    { display: "Dummy Jens Jensen" },
+                    { display: "Dummy Some Corporation" }
+                  ],
+                  manifestations: { first: { pid: "870970-basis:25245784" } }
+                }
+              },
+              {
+                type: "CREATOR",
+                term: "Yuval Noah Harari",
+                work: {
+                  workId: "work-of:870970-basis:51725832",
+                  titles: { main: ["Dummy Some Title"] },
+                  creators: [
+                    { display: "Dummy Jens Jensen" },
+                    { display: "Dummy Some Corporation" }
+                  ],
+                  manifestations: { first: { pid: "870970-basis:51725832" } }
+                }
+              },
+              {
+                type: "TITLE",
+                term: "Harry Potter and the philosopher's stone",
+                work: {
+                  workId: "work-of:870970-basis:42042455",
+                  titles: { main: ["Dummy Some Title"] },
+                  creators: [
+                    { display: "Dummy Jens Jensen" },
+                    { display: "Dummy Some Corporation" }
+                  ],
+                  manifestations: { first: { pid: "870970-basis:42042455" } }
+                }
+              }
+            ]
+          }
+        }
+      }
+    });
+    cy.intercept("GET", "**/covers**", {
+      statusCode: 200,
+      body: [
+        {
+          id: "870970-basis:25245784",
+          type: "pid",
+          imageUrls: {
+            small: {
+              url: "https://res.cloudinary.com/dandigbib/image/upload/t_ddb_cover_small/v1543886150/bogportalen.dk/9788702029444.jpg",
+              format: "jpeg",
+              size: "small"
+            }
+          }
+        }
+      ]
+    });
+    cy.visit("/iframe.html?args=viewMode=story&id=apps-search-header--default");
+  });
+
+  it("Allows user to write into the search field", () => {
+    cy.get(".header__menu-search-input")
+      .should("be.visible")
+      .focus()
+      .type("ad");
+  });
+
+  it("Doesn't show suggestions before 3 characters are typed", () => {
+    cy.get(".header__menu-search-input").focus().type("ha");
+    cy.get(".autosuggest").should("not.be.visible");
+    cy.get(".header__menu-search-input").focus().type("r");
+    cy.get(".autosuggest").should("be.visible");
+  });
+
+  it("Allows use of arrow keys to navigate autosuggest", () => {
+    cy.get(".header__menu-search-input").focus().type("harry");
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.wait(1000);
+    cy.get(".header__menu-search-input").focus().type("{downArrow}");
+    cy.get("#downshift-0-item-0").should("have.attr", "aria-selected", "true");
+  });
+
+  it("Matches text in the search field with highlighted item", () => {
+    cy.get(".header__menu-search-input").focus().type("har");
+    // eslint-disable-next-line cypress/no-unnecessary-waiting
+    cy.wait(1000);
```

Comment:
I am still in doubt wether it is not possible to solve this with .then instead. Have you investigated that?
https://docs.cypress.io/api/commands/then#Syntax

Reply:
Kasper found a nice way to make it work and not use wait :)